railties/test/application/configuration_test.rb

```diff
@@ -1194,6 +1194,39 @@ def create
       assert_equal false, ActionController::Parameters.action_on_unpermitted_parameters
     end
 
+    test "config.action_controller.permit_all_parameters can be configured in an initializer" do
+      app_file "config/initializers/permit_all_parameters.rb", <<-RUBY
+        Rails.application.config.action_controller.permit_all_parameters = true
+      RUBY
+
+      app "development"
+
+      ActionController::Base.object_id # force lazy load hooks to run
```

Comment:
This is pretty ugly, but I couldn't come up with a better way to trigger the lazy load hooks.

Just referencing `ActionController::Base` without calling a method on it resulted in this warning:

```
railties/test/application/configuration_test.rb:1204: warning: possibly useless use of :: in void context
```